src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/GoToPosition.java

```diff
@@ -3,25 +3,50 @@
 import edu.greenblitz.tobyDetermined.RobotMap;
 import edu.greenblitz.tobyDetermined.commands.telescopicArm.elbow.RotateToAngle;
 import edu.greenblitz.tobyDetermined.commands.telescopicArm.extender.ExtendToLength;
-import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Elbow;
-import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Extender;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 
 public class GoToPosition extends SequentialCommandGroup {
 
     public GoToPosition(double lengthInMeters, double angleInRads) {
```

Comment:
logic is inside the constructor should use command group logic if need be can be split to multiple commands and built using conditionalCommand

Reply:
have you heard of DIY?